core/controllers/profile.py

```diff
@@ -368,8 +372,18 @@ def get(self):
         if not constants.ENABLE_ACCOUNT_EXPORT:
             raise self.PageNotFoundException
 
-        user_data = takeout_service.export_data_for_user(self.user_id)
-        self.render_json(user_data)
+        user_data, user_images = takeout_service.export_data_for_user(
```

Comment:
I think you should have a simple domain object for a "user_image", since it seems to be more than just an image.

In general, if you are passing objects around, try and actually make them objects, rather than tuples/dicts. Otherwise it gets confusing because you then start referring to `image[0]` and the reader has no idea what that is.

Reply:
Done